src/test/java/com/google/cloud/spanner/pgadapter/ITJdbcTest.java

```diff
@@ -166,24 +166,29 @@ public void testSelectHelloWorld() throws SQLException {
 
   @Test
   public void testSelectWithParameters() throws SQLException {
```

Comment:
Why are we doing a JDBC test in pgAdapter (that is what I got from the test name, but maybe I am wrong)?

Reply:
You assumption is correct, but what we are doing here is an end-to-end test using the native PG JDBC driver to ensure that PGAdapter actually behaves the same as PostgreSQL. So the test uses the native PG JDBC driver and connects to PGAdapter, which again is connected to Cloud Spanner. Queries/DML statements etc. are then sent from the native PG driver through PGAdapter to Spanner and then back through the same channel to verify that PGAdapter behaves the same as PostgreSQL would do.

So basically the same as our existing psql end-to-end tests, but now using the PG JDBC driver.